docs/build-configs.md

```diff
@@ -0,0 +1,76 @@
+# Build Configs
+
+The concept of build configs is used to build different versions of the app. A build config defines the following (among others):
+* Enabling and disabling features
+* Design, fonts, icons and theme colors
+* CMS Urls
+* Override locales
+* Store metadata
+
+## Available Build Configs
+
+* **integreat-test-cms**: Build config for development, very similar to integreat but using the [test cms](technical-documentation.md#test-cms).
+* **integreat**: Integreat build config using the production integreat cms.
+* **malte**: Build config for the Malte-App which uses a different design, name and cms.
+* **integreat-e2e**: Build config for [E2E-Tests](e2e-testing.md), primarily used in the [CI](cicd.md).
+
+**NOTE: Testing and developing with the live cms instance should be avoided. Therefore, the `integreat-test-cms` build config should be used primarily.**
+
+## Using a Build Config
+
+Build configs are used in two different places: In the javascript code during runtime and in gradle and XCode during the build process.
+Therefore, the build config has to be defined when running the bundler AND when building the app.
+
+### Runtime (Javascript)
+
+For each build config there is a script in the [package.json](../package.json) to ease the process of starting the packager:
+```bash
+yarn start:<build config name>
+```
+
+For the standard development build config `integreat-test-cms` there is also the shortcut `yarn start`.
+
+To access the values of the build config use [this method](../src/modules/app/constants/buildConfig.js).
+
+### Gradle (Android Build)
+
+For each build config there is a script in the [package.json](../package.json) to ease the process of building and installing the app on android:
+```bash
+yarn android:<build config name>
+```
+
+For the standard development build config `integreat-test-cms` there is also the shortcut `yarn android`.
+
+To build `integreat` and `malte` in **production** mode, use the commands `yarn android:integreat:production` and `yarn android:malte:production`.
+
+### XCode (iOS Build)
+
+To use a build config in XCode, select the corresponding xcscheme and hit run or archive as specified in the [manual builds doc](manual-builds.md#ios).
+Before running the app, make sure to start the packager with the right build config as specified [above](#runtime-javascript)
+as the packager started by XCode per default does not select a build config.
```

Comment:
Maybe we can add the error message if this is not done here

Reply:
Done